labs/coding-102-rest-python-ga/2.md

```diff
@@ -10,7 +10,7 @@ import requests
 import json
 
 # Put the IP address or DNS of your DNA Center in this URL
-url = 'https://{APIC-EM-Controller}/api/v1/ticket'
+url = 'https://{DNA-Center-Server}/api/v1/ticket'
```

Comment:
When we switch out code examples, this won't be ticket any more.

Reply:
Okay. We'll discuss the code example changes with you.